app/scripts/modules/core/src/application/nav/CategoryDropdown.tsx

```diff
@@ -55,7 +60,7 @@ export class CategoryDropdown extends React.Component<ICategoryDropdownProps, IC
       });
     this.entityTagsSubscription = merge(...category.dataSources.map(ds => ds.refresh$))
       .subscribe(() => {
-        const tags = category.dataSources.reduce((acc: IEntityTags[], ds: ApplicationDataSource) => acc.concat(ds.alerts), []);
+        const tags = category.dataSources.reduce((acc: IEntityTags[], ds: ApplicationDataSource) => acc.concat(ds.alerts || []), []);
```

Comment:
an error was being thrown here because we'd sometimes concat an `undefined` into the array.